### PR TITLE
Updated geo-location.html to Polymer 2.x

### PR DESCRIPTION
**Pull request by sheideman**
Here is the update I promised to Polymer 2.x Eric. I tested it and it works as well as the 1.x version did.

### CODE REVIEW

File: geo-location.html [hunk 1]
```diff
@@ -1,217 +1,174 @@
...
-<!-- TODO: decide if we need to cache results across instances. -->
-
-<!--
-Provides the current location using the [Geolocation API](https://developer.mozilla.org/en-US/docs/Web/API/Geolocation/Using_geolocation).
-
-<b>Example - basic</b>
-
-    <geo-location></geo-location>
-
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
```
Comment (ebidel) [inline geo-location.html]: - Remove "bower_components' from the import.
- Remove the commented out elements.

File: geo-location.html [hunk 2]
```diff
@@ -1,217 +1,174 @@
-<!-- Copyright Eric Bidelman <ebidel@gmail.com> -->
```
Comment (ebidel) [inline geo-location.html]: Let's keep this :)

File: geo-location.html (hunk 1 above)
Comment (ebidel) [inline geo-location.html]: Can you keep double quotes for html attributes?

File: geo-location.html [hunk 3]
```diff
@@ -1,217 +1,174 @@
...
-<b>Example - basic</b>
-
-    <geo-location></geo-location>
-
+<link rel='import' href='../bower_components/polymer/polymer-element.html'>
+<!--<link rel='import' href='../bower_components/polymer/lib/elements/dom-if.html'>-->
+<!--<link rel='import' href='../bower_components/polymer/lib/elements/dom-repeat.html'>-->
+<dom-module id='geo-location'>
     <script>
-      var loc = document.querySelector('geo-location');
```
Comment (ebidel) [inline geo-location.html]: Want to move this stuff to the README so it gets picked up by webcomponents.org? You can follow https://www.webcomponents.org/publish.

File: geo-location.html [hunk 4]
```diff
@@ -1,217 +1,174 @@
...
-         * If true, enables high accuracy GPS.
-         */
-        highAccuracy: {
-          type: Boolean,
-          value: false
-        },
+        /** @polymerElement */
+        class GeoLocation extends Polymer.Element {
+            static get is() {
+                return 'geo-location';
```
Comment (ebidel) [inline geo-location.html]: 2 space indent please

File: geo-location.html (hunk 2 above)
Comment (sheideman) [inline geo-location.html]: You got it boss! :)

File: geo-location.html (hunk 1 above)
Comment (sheideman) [inline geo-location.html]: Done & Done. sorry--was testing before I put it up.

Comment (sheideman) [inline geo-location.html]: Done.

File: geo-location.html (hunk 3 above)
Comment (sheideman) [inline geo-location.html]: I think I screwed this up. I will read the webcomponents.org/publish specs and make the necessary adjustments.

File: geo-location.html (hunk 4 above)
Comment (sheideman) [inline geo-location.html]: 2 spaces??!!! Blasphemy! ;) Just kidding. Updated.

**Comment by FluorescentHallucinogen** [comment]
Shouldn't `bower.json` be updated too?

**Comment by sheideman** [comment]
sorry guys. Stuck in project code hermit mode. Closing.

**Pull request closed without merging**